SPECIFICATION.md

```diff
@@ -312,6 +331,7 @@ TODO
 | license | The SPDX identifier for the license of this font. Create multiple `license` entries if the font is published under multiple licenses. | 1-n | `Apache-2.0` |
 | letter_spacing | Global letter space width for every glyph. Same unit as for polylines. If not specified, 0 is assumed (no space). | 0-1 | `1.8` (= 20% of font height) |
 | line_spacing | Vertical baseline distance of multiline texts. Same unit as for polylines. If not specified, 9 is assumed (no space). | 0-1 | `16` (= 178% of font height) |
+| monospace_width | Specifying this header marks the font as a monospace font. The value specifies the width of all glyphs (excluding global spacing). | 0-1 | `9` (= 100% of font height) |
```

Comment:
I think 9 is too wide for a typical monospace font. Probably 6 or 7 would be more realistic.

Reply:
That's just an example, right? But if you want I can adjust it to 6.

Reply:
Yes it's just an example, but the example values from the other keys are realistic for a typical font, so I'd like to see here also a realistic value.